content/docs/0.14.x/reference/files/sli-provider/index.md

```diff
@@ -1,15 +1,17 @@
 ---
 title: SLI-Provider
```

Comment:
I'm questioning why this page is in the `Reference/Files` section. I would rather see it in `Quality Gates` because it is not a file but needed for a Quality Gate: 

![image](https://user-images.githubusercontent.com/729071/162143034-94a28cb6-8919-41bf-82bf-49cbcc9b8a72.png)

Reply:
I rather agree.  It got moved over with the others but I'm going to move it back.

Reply:
Done